examples/Plantfarm.hs

```diff
@@ -209,14 +213,14 @@ main = do
       S.json ps
 
 handleWebError :: Either Err b -> S.ActionM b
-handleWebError (Left e) = S.raise (TL.pack $ "An error occurred:\n" <> show e)
+handleWebError (Left e) = error $ "An error occurred:\n" <> show e
```

Comment:
I suppose you're replacing this because `raise` is deprecated. Wouldn't it be better to migrate `throw` which isn't deprecated? Maybe it's better to do this in a separate PR and leave `raise` alone for now, since it is still in the latest scotty. You can disable the warning for the module using OPTIONS_GHC.

Reply:
That's a fair assessment. The motivation *was* simply to suppress the "deprecation" warning. I do concur that switching from `raise` to `throw`should be a separate PR. With this in mind I have reverted this change.